### PR TITLE
chore: update contributing

### PR DESCRIPTION
**Pull request by ameter**
*Description of changes:*
update contributing

*Check points: (check or cross out if not relevant)*

- ~~[ ] Added new tests to cover change, if needed~~
- ~~[ ] Build succeeds with all target using Swift Package Manager~~
- ~~[ ] All unit tests pass~~
- ~~[ ] All integration tests pass~~
- [x] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- ~~[ ] Documentation update for the change if required~~
- [x] PR title conforms to conventional commit style
- ~~[ ] If breaking change, documentation/changelog update with migration instructions~~

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,92 +29,40 @@ For a brief history and our ethos/design philosophy around the Amplify project p
 
 ## Getting Started
 
-To get started with the Amplify Framework for iOS, first make sure you have the latest version of Xcode installed as well as cocoapods. You can install cocoapods by simply running:
-
-```
-sudo gem install cocoapods
-```
+To get started with the Amplify Library for Swift, first make sure you have the latest version of Xcode installed.
```
Comment (thisisabhash) [inline CONTRIBUTING.md:32]: should we add the min XCode version supported?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -157,7 +105,7 @@ Create a commit with the proposed change changes:
 
 - Push your changes to your GitHub fork
 - Submit a Pull Requests on the amplify-ios repo to the `main` branch and add the Amplify Native team using @aws-amplify/amplify-native so we can approve/provide feedback.
```
Comment (thisisabhash) [inline CONTRIBUTING.md:107]: amplify-swift?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -131,7 +79,7 @@ The GitHub issue is sufficient for such discussions, and can be sufficient to ge
 First, create a fork of amplify-ios. Clone it, and make changes to this fork.
```
Comment (thisisabhash) [inline CONTRIBUTING.md:79]: amplify-swift?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ameter) [reply CONTRIBUTING.md:32]: good call.  done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ameter) [reply CONTRIBUTING.md:107]: good catch.  done

File: CONTRIBUTING.md (hunk 3 above)
Comment (ameter) [reply CONTRIBUTING.md:79]: yes, done.

**Review by thisisabhash — APPROVED** [review]

**Pull request merged**